cranelift/codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1600,13 +1600,19 @@ fn define_simd(
     let regspill = shared.by_name("regspill");
     let sadd_sat = shared.by_name("sadd_sat");
     let scalar_to_vector = shared.by_name("scalar_to_vector");
+    let sload8x8 = shared.by_name("sload8x8");
```

Comment:
It might be clearer that these are extending loads by adding an 'x' to the name, viz, `sxloadNxM` and `uxloadNxM`.  I think it would be nice to make that explicit.  But I don't know if that makes them inconsistent with existing scalar extending load insns (if indeed such things even exist).

Reply:
I think you are right, it would be more explicit but that would be inconsistent with `sload8`, `uload16`, etc.